internal/app/machined/pkg/controllers/network/link_spec_test.go

```diff
@@ -371,24 +372,30 @@ func (suite *LinkSpecSuite) TestBond() {
 	dummy0Name := suite.uniqueDummyInterface()
 	dummy0 := network.NewLinkSpec(network.NamespaceName, dummy0Name)
 	*dummy0.TypedSpec() = network.LinkSpecSpec{
-		Name:        dummy0Name,
-		Type:        nethelpers.LinkEther,
-		Kind:        "dummy",
-		Up:          true,
-		Logical:     true,
-		MasterName:  bondName,
+		Name:    dummy0Name,
+		Type:    nethelpers.LinkEther,
+		Kind:    "dummy",
+		Up:      true,
+		Logical: true,
+		BondSlave: network.BondSlave{
+			MasterName: bondName,
+			SlaveIndex: 0,
+		},
 		ConfigLayer: network.ConfigDefault,
 	}
 
 	dummy1Name := suite.uniqueDummyInterface()
 	dummy1 := network.NewLinkSpec(network.NamespaceName, dummy1Name)
 	*dummy1.TypedSpec() = network.LinkSpecSpec{
-		Name:        dummy1Name,
-		Type:        nethelpers.LinkEther,
-		Kind:        "dummy",
-		Up:          true,
-		Logical:     true,
-		MasterName:  bondName,
+		Name:    dummy1Name,
+		Type:    nethelpers.LinkEther,
+		Kind:    "dummy",
+		Up:      true,
+		Logical: true,
+		BondSlave: network.BondSlave{
+			MasterName: bondName,
+			SlaveIndex: 0,
```

Comment:
```suggestion
			SlaveIndex: 1,
```

So that the order of enslaving is deterministic?

Reply:
Done.